_data/infrastructure-projects/police-stations-hash147.yaml

```
@@ -0,0 +1,34 @@
+name: Police Stations
+coordinates:
+  - -30.4759, 30.2030
+stage: Various
+total_budget: 142200000000
+department: Police
+description: Construction of new and re-established police stations
+projected_budget: 51800000000
+department_link: /path/to/department/detail/page
+nature_of_investment: Upgrading and additions
+infrastructure_type: Police stations
+google_maps_link: https://www.google.com/maps/@-30.4759,30.2030,17z
+data:
```

let's make this name a bit more meaningful
```suggestion
expenditure:
```